test/unit/liquidity_pool_endpoints_test.js

```diff
@@ -141,4 +141,492 @@ describe('/liquidity_pools tests', function() {
       });
     });
   });
+
+  describe('querying a specific pool', function() {
+    const lpId = "ae44a51f6191ce24414fbd1326e93ccb0ae656f07fc1e37602b11d0802f74b9a";
+
+    const poolOpsResponse = {
+      "_links": {
+        "self": {
+          "href": "https://private-33c60-amm3.apiary-mock.com/liquidity_pools/1/operations?cursor=113725249324879873&limit=10&order=asc"
```

Comment:
nit: I know these are only smoking tests but the links LP ID should be in these links too:
```suggestion
          "href": "https://private-33c60-amm3.apiary-mock.com/liquidity_pools/ae44a51f6191ce24414fbd1326e93ccb0ae656f07fc1e37602b11d0802f74b9a/operations?cursor=113725249324879873&limit=10&order=asc"
```

Reply:
Great point :+1: